src/helm/benchmark/metrics/vision_language/image_utils.py

```diff
@@ -0,0 +1,124 @@
+import warnings
+from scipy.stats import wasserstein_distance
+import numpy as np
+import cv2
+from PIL.Image import Image
```

Comment:
These need to be handled via `handle_module_not_found_error`

Reply:
Done